File: tests/fixtures/languages/sample.clj
;; Minimal Clojure fixture for language parse-validation tests.

(ns demo.core
  (:require [clojure.string :as str]))

(defn greet [label]
  (str "Hello, " label))

(defn add [a b]
  (+ a b))

(def default-label "world")

File: tests/fixtures/languages/sample.hs
-- Minimal Haskell fixture for language parse-validation tests.

module Demo where

import Data.List (sort)

data Greeter = Greeter { label :: String }

greet :: Greeter -> String
greet g = "Hello, " ++ label g

add :: Int -> Int -> Int
add a b = a + b
